To: vim_dev@googlegroups.com
Subject: Patch 7.3.266
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.266
Problem:    In Gvim with iBus typing space in Insert mode doesn't work.
Solution:   Clear xim_expected_char after checking it.
Files:	    src/mbyte.c


*** ../vim-7.3.265/src/mbyte.c	2011-07-15 21:16:54.000000000 +0200
--- src/mbyte.c	2011-07-27 18:21:13.000000000 +0200
***************
*** 5170,5175 ****
--- 5170,5179 ----
  		/* We had a keypad key, and XIM tried to thieve it */
  		return FALSE;
  
+ 	    /* This is supposed to fix a problem with iBus, that space
+ 	     * characters don't work in input mode. */
+ 	    xim_expected_char = NUL;
+ 
  	    /* Normal processing */
  	    return imresult;
  	}
*** ../vim-7.3.265/src/version.c	2011-07-27 17:58:42.000000000 +0200
--- src/version.c	2011-07-27 18:23:00.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     266,
  /**/

-- 
    [clop clop]
ARTHUR:  Old woman!
DENNIS:  Man!
ARTHUR:  Man, sorry.  What knight lives in that castle over there?
DENNIS:  I'm thirty seven.
ARTHUR:  What?
DENNIS:  I'm thirty seven -- I'm not old!
                                  The Quest for the Holy Grail (Monty Python)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
